### ISSUE TITLE
Can't disable karma automatism

### ISSUE THREAD

**Issue by lkundrak** [fedora-infra/bodhi#434]
$SUBJ

Does not work. At all.

**Comment by ralphbean** [comment]
There's not enough detail in this bug report to be useful.

* Did you try the web UI?
* Did you try the client?
* Did the checkbox change, but not the behavior?
* Which update(s) did you try?
* Were they submitted before or after the bodhi2 migration last wednesday?

Don't be rude.

**Comment by jwboyer** [comment]
I'm having trouble with this as well.  Hopefully the info below helps.

* I filed this update with the web UI a few minutes ago: https://bodhi.fedoraproject.org/updates/kernel-4.2.0-1.fc23
* I did not try the client
* I unchecked the "Auto-request stable" checkbox when I filed the update
* When the update was submitted, it still has karma values of 3, -3
* When I click edit, the checkbox for "Auto-request stable" is checked

So it appears that no matter what I did with the checkbox, I cannot disable autokarma.  This kind of breaks the workflow the kernel team uses, which is to leave autokarma off and let karma collect over a period of time either way.  We then determine based on feedback when to push to stable.
